frappe/core/doctype/error_log/error_log.js

```diff
@@ -12,6 +12,9 @@ frappe.ui.form.on("Error Log", {
 					reference_name: frm.doc.reference_name,
 				});
 			});
+			frm.add_custom_button(__(frm.doc.reference_name), function () {
```

Comment:
```suggestion
			frm.add_custom_button(__("Open reference document"), function () {
```
because a button with random ID will not make much sense

Reply:
This is incorpodated in https://github.com/frappe/frappe/pull/26817/commits/1c3a5c740358d6b3467ae7bb7fc9fea5fbb8daf1